packages/yoroi-extension/app/components/wallet/navigation/WalletNavButton.js

```diff
@@ -35,7 +36,11 @@ export default class WalletNavButton extends Component<Props> {
       <button type="button" className={componentClasses} onClick={onClick}>
         {IconComponent != null &&
           <div className={styles.icon}>
-            <IconComponent />
+            {
+              label === 'Transactions' && isActive ?
+              <TransactionImageActive />
+              : <IconComponent />
+            }
```

Comment:
@AhmedIbrahim336 , this is not a great way to add some logic:

1. A component like `WalletNavButton` is very abstract and should not be aware of a specific icon like `TransactionImageActive`. It works only with generic properties and should be kept this way.

2. You cannot use label for a hardcoded check to decide which tab is it. Not only your logic will only work if no one ever changes the text, but also all labels we have are `intl` rendered strings, which means that it will be different for different languages.

You need to change this. This logic must be moved to another place where the different tabs are actually handled. A button shouldn't do that.

Reply:
I got you point 
so the button is a reusable component it should have this logic in it.
Also this will not work with different languages and when we change the text.
Okay I will try to think for another way to do it